### ISSUE TITLE
YGOPro version control

### ISSUE THREAD

**Issue by garymabin** [Fluorohydride/ygopro#1666]
I see there are many version updates recently, and IFC changes keep bringing in before we finally update the version codes.  I know complex branching strategy is not suitable for this project, but can we at least add a tag to each updated version so that every derivative version could share the same baseline with certain commit?

**Comment by Zayelion** [comment]
I agree, and now that the core and scripts are seperate its really just the gframe in this repo. It should be tagged.
